### PR TITLE
Adding requirement to README.md, and package.json

### PR DESCRIPTION
**Pull request by Omcsesz**

### CODE REVIEW

**Comment by marstamm** [comment]
Hi, thanks for opening this PR.

If this is related to an issue you had, do you think it would it help to specify the `engine` also in the `package.json`?

**Comment by Omcsesz** [comment]
> Hi, thanks for opening this PR.
> 
> If this is related to an issue you had, do you think it would it help to specify the `engine` also in the `package.json`?

I think you're right.

File: package.json [hunk 1]
```diff
@@ -31,5 +31,9 @@
     "eslint-plugin-bpmn-io": "^0.10.0",
     "mocha": "^6.2.2",
     "npm-run-all": "^4.1.5"
+  },
+  "engines": {
+    "node": ">= 18.0.0",
+    "npm": ">= 8.0.0"
```
Comment (marstamm) [inline package.json:37]: We still test this one on Node 10, so don't have to be as restrictive with the version numbers

```suggestion
    "node": ">= 10.0.0",
    "npm": ">= 6.0.0"
```

**Review by marstamm — APPROVED** [review]
Looks good, let's merge it

Thank you for your contribution!

**Pull request merged**